src/LodeRunner.API.Test/IntegrationTests/HttpClientExtension.cs

```diff
@@ -160,5 +164,177 @@ public static async Task<bool> WaitAndValidateGetClientsToMatchId(this HttpClien
 
             return found;
         }
+
+        /// <summary>
+        /// Get Test Runs.
+        /// </summary>
+        /// <param name="httpClient">the httpClient.</param>
+        /// <param name="getTestRunsUri">getTestRuns Uri.</param>
+        /// <param name="jsonOptions">The json options.</param>
+        /// <param name="output">The output.</param>
+        /// <returns>the task.</returns>
+        public static async Task<bool> GetTestRuns(this HttpClient httpClient, string getTestRunsUri, JsonSerializerOptions jsonOptions, ITestOutputHelper output)
```

Comment:
It looks like some of the response code validation that I was looking for in the test is done in the extension.  Is there a reason some of it is in the extension versus the test method.  My starting expectation is that all validation would happen in the test method.

Reply:
moved to test method